src/pc/protocols/usb_host_ep.cpp

```diff
@@ -0,0 +1,115 @@
+// project
+#include <cstdlib>
+#define MVLOG_UNIT_NAME xLinkUsb
+
+#include "XLink/XLinkLog.h"
+#include "XLink/XLinkPlatform.h"
+#include "XLink/XLinkPublicDefines.h"
+#include "usb_host_ep.h"
+#include "../PlatformDeviceFd.h"
+
+#if not defined(_WIN32)
+#include <unistd.h>
+#include <fcntl.h>
+#endif
+
+#include <stdlib.h>
+#include <cstring>
+
+static int usbFdRead, usbFdWrite;
+
+int usbEpInitialize() {
+    return 0;
+}
+
+int usbEpPlatformConnect(const char *devPathRead, const char *devPathWrite, void **fd)
+{
+    return X_LINK_ERROR;
+}
+
+int usbEpPlatformServer(const char *devPathRead, const char *devPathWrite, void **fd)
+{
+#if defined(_WIN32)
+    return X_LINK_ERROR;
+#else
+    char outPath[256];
+    strcpy(outPath, devPathWrite);
+    strcat(outPath, "/ep1");
+
+    char inPath[256];
+    strcpy(inPath, devPathWrite);
+    strcat(inPath, "/ep2");
+
+    int outfd = open(outPath, O_RDWR);
+    int infd = open(inPath, O_RDWR);
+
```

Comment:
c++ std::string instead

+add a special case where devPathREAD is supplied as well and then don't do any "auto /ep1&2 assumptions" but just directly open outPath & open inPath and use that as FD's